code/datums/diseases/transformation.dm

```diff
@@ -127,9 +127,11 @@
 	if(affected_mob.mind && !is_monkey(affected_mob.mind))
 		add_monkey(affected_mob.mind)
 	if(ishuman(affected_mob))
-		if(affected_mob && !is_monkey_leader(affected_mob.mind))
-			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)
+		if(affected_mob && !is_monkey_leader(affected_mob.mind) && !prob(4))
```

Comment:
Why are we doing `!prob(4)` instead of `prob(96)`

Reply:
So you can input the probability rate (4%) instead of 96% (100-4). It's the same.